x-pack/plugins/security/public/account_management/change_password/change_password_async.tsx

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+
+import type { CoreStart } from 'src/core/public';
+
+import { UserAPIClient } from '../../management/users';
+import type { ChangePasswordProps } from './change_password';
+
+export const getChangePasswordComponent = async (
+  core: CoreStart
+): Promise<React.FC<Pick<ChangePasswordProps, 'user'>>> => {
```

Comment:
__nit:__ I learnt yesterday that [`Pick` doesn't play well with our dev docs](https://github.com/elastic/kibana/pull/101661#discussion_r648260136). Would you mind creating two separate interfaces inside of `change_password` instead and replacing any `Pick`-based types with dedicated interface?

```ts
export interface ChangePasswordProps {
  user: AuthenticatedUser;
}

export interface ChangePasswordPropsInternal extends ChangePasswordProps {
  userAPIClient: PublicMethodsOf<UserAPIClient>;
  notifications: NotificationsSetup;
}
```

(and you also need to merge the latest master and likely to update latest version of `SecurityPluginStart`)

Reply:
Ah, good to know, and thanks for the link! I replaced it here: https://github.com/elastic/kibana/pull/99791/commits/24258211021c0b995ff548521a172acaac11c636